data/malicodes2.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Muhammad Ali",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "SE Undergrad | OpenSource Enthusiast | Freelance Web Developer ",
+  "avatar": "https://github.com/malicodes2.png",
+  "tags": ["React JS", "JavaScript", "Front-end Developer", "Open Source"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/malicodes2" },
+    { "icon": "SiHashnode", "url": "https://malicodes.hashnode.dev/" },
+    { "icon": "FaGithub", "url": "https://github.com/malicodes2" }
+  ],
+  "links": [
+    {
+      "name": "GitHub",
+      "url": "https://github.com/malicodes2",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/malicodes2",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/malicodes",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Blogs",
+      "url": "https://malicodes.hashnode.dev/",
+      "icon": "SiHashnode"
+    }
+  ]
```

Comment:
Hey you're missing a `}` at the end

Reply:
Adding...!

Reply:
Done

Reply:
The `}` on line 34 needs to be moved after the `]` on line 35

Reply:
Oh I apologize. Thanks for pointing out.